tests/test_geomopt_cli.py

```diff
@@ -0,0 +1,144 @@
+"""Test geomopt commandline interface."""
+
+from pathlib import Path
+
+from ase import Atoms
+from ase.io import read
+from typer.testing import CliRunner
+
+from janus_core.cli import app
+
+DATA_PATH = Path(__file__).parent / "data"
+
+runner = CliRunner()
+
+
+def test_geomopt_help():
+    """Test calling `janus geomopt --help`."""
+    result = runner.invoke(app, ["geomopt", "--help"])
+    assert result.exit_code == 0
+    # Command is returned as "root"
+    assert "Usage: root geomopt [OPTIONS]" in result.stdout
+
+
+def test_geomopt():
+    """Test geomopt calculation."""
+    results_path = Path(".").absolute() / "Cl4Na4-opt.xyz"
+    assert not results_path.exists()
+
+    result = runner.invoke(
+        app,
+        [
+            "geomopt",
+            "--struct",
+            DATA_PATH / "NaCl.cif",
+            "--max-force",
+            "0.2",
+        ],
+    )
+    assert result.exit_code == 0
+    atoms = read(results_path)
+    assert isinstance(atoms, Atoms)
+    results_path.unlink()
```

Comment:
This will never be reached if the read fails leaving the file extant. Probably want a `with Tempfile` or similar

Reply:
Maybe it's not an essential test, but part of the point of this was to test the default options, which includes a pre-defined path for the optimized structure.

Every other test does use temporary files, but I'm not sure there's a way to do that for this, unless I'm misunderstanding how Tempfile would work?

Reply:
I mean, the only obvious way to do this would be:
```
    try:
        assert result.exit_code == 0
        atoms = read(results_path)
        assert isinstance(atoms, Atoms)
    finally:
        results_path.unlink()

Reply:
we were chatting exactly on that line but maybe with a try only around reading